src/components/Common/VideoCard/VideoOptions.tsx

```diff
@@ -83,11 +83,11 @@ const VideoOptions = ({
           </button>
           <button
             type="button"
-            onClick={() =>
+            onClick={() => {
```

Comment:
can you quick function in the same file?

Reply:
Can you please elaborate that?
The issue was that arrow function was written along with ternary which could be confusing so I added braces as the possible solution suggested by deepsource.io

Reply:
What you did is good. It can be a function that does the same logic rather than having it inside jsx.

Reply:
something like `onClick={() => onClickWatchLater()}`

Reply:
Sure i would do it